src/inspector_agent.cc

```diff
@@ -46,9 +58,14 @@ std::unique_ptr<StringBuffer> ToProtocolString(Isolate* isolate,
 
 // Called from the main thread.
 void StartInspectorIoThreadAsyncCallback(uv_async_t* handle) {
-  static_cast<Agent*>(handle->data)->StartIoThread();
+  static_cast<Agent*>(handle->data)->StartIoThread(false);
+}
+
+void StartIoCallback(Isolate* isolate, void* agent) {
```

Comment:
That's "InterruptCallback" typedef - https://github.com/nodejs/node/blob/master/deps/v8/include/v8.h#L5994